project/Dependencies.scala

```diff
@@ -0,0 +1,30 @@
+import sbt._
+
+object Dependencies {
+  val akkaV = "2.5.1"
+  val akkaHttpV = "10.0.6"
```

Comment:
Since when did akka-http jump from 2.x to 10.x!?

Anyway looks like the latest is 10.0.9: http://doc.akka.io/docs/akka-http/current/scala/http/index.html